docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -1489,6 +1489,7 @@ This section contains reference information about each of the configuration prop
 |`fhirServer/bulkdata/cosFileMaxResources`|int|The maximum number of FHIR resources per COS file, "-1" means no limit, the default value is 500000 |
 |`fhirServer/bulkdata/cosFileMaxSize`|int|The maximum COS file size in bytes, "-1" means no limit, the default value is 209715200 (200M) |
 |`fhirServer/bulkdata/patientExportPageSize`|int| The search page size for patient/group export, the default value is 200 |
+|`fhirServer/bulkdata/isCosClientUseFhirServerTrustStore`|boolean| If use fhir server TrustStore to access S3/IBMCOS service |
```

Comment:
```suggestion
|`fhirServer/bulkdata/isCosClientUseFhirServerTrustStore`|boolean| If the COS Client should use the IBM FHIR Server's TrustStore to access S3/IBMCOS service |
```

Separately, this property is pretty long, it might be best to summarize it to a short property and easier to understand. 

`isCosClientUseFhirServerTrustStore`
-> 
`useDefaultTrustStore`

Reply:
make sense, let  me change it together with the doc.